VISION.md

```diff
@@ -19,7 +19,7 @@ Instead, MessageKit will provide you with hooks to easily handle your different
 We will strive to support the 3 latest versions of iOS.
 
 - **Objective-C Compatability**: 
-We will not sacrifice functionality or an idiomatic Swift API to support Objective-C.
+We will not sacrifice functionality or an idiomatic Swift API to support Objective-C, but would love to improve Objective-C Compatability where possible.
 
 - **Layouts**: 
-We will favor programmatic layouts over Xibs where ever possible.
+We will favor programmatic layouts over `.xib`s where ever possible.
```

Comment:
Can we add a reasoning behind this maybe?

Reply:
Yeah, I'm pretty biased about storyboards/xibs. They're slow (parsing XML -> constraints) and prone to merge conflict. I'd rather do the dirty work and provide a better final product to MessageKit users